client/scss/components/_button.scss

```
@@ -119,7 +119,7 @@
       position: absolute;
       inset-inline-start: 0;
       top: 0;
-      width: 3em;
+      width: 18%;
```

This would be problematic because then the icon width is dependent on the button width.

```suggestion
      width: 3em;
```